Gemfile

```
@@ -8,3 +8,4 @@ gemspec
 gem "rake", "~> 13.0"
 
 gem "rspec", "~> 3.0"
+gem "base64" if RUBY_VERSION >= "3.4.0"
```

RuboCop (1.75.5)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `base64` should appear before `rspec`.

(Bundler/OrderedGems)